variables.tf

```diff
@@ -86,8 +87,26 @@ variable "fifo_topic" {
   default     = false
 }
 
+variable "fifo_queue" {
```

Comment:
```suggestion
variable "fifo_queue_enabled" {
```

Our convention/pattern

Reply:
the same for `fifo_topic_enabled`

Reply:
Since fifo_topic is the same attribute as aws_sns_topic, shouldn't we keep that consistent ?

Reply:
let's keep it